server/src/providers/wallets.ts

```diff
@@ -198,16 +198,16 @@ class Wallets {
       let key = `${ceilFunc(minStep+1)}-${maxStep}`
 
       const filter = {
-        [field]:{ $gt: minStep, $lte: maxStep }
+        [field]:{ $gte: minStep, $lte: maxStep }
       }
       if (j===0) {
-        filter[field] = { $gt: 0, $lte: maxStep }
-        key = `${ceilFunc(min+1)}-${maxStep}`
+        filter[field] = { $gte: 0, $lte: maxStep }
+        key = `${ceilFunc(min)}-${maxStep}`
```

Comment:
why '+1' was removed here but kept in 'minStep+1' ?

Reply:
it shouldn't be removed as the $gte used only for the first step. See new commits